src/cross_toml.rs

```diff
@@ -189,17 +184,18 @@ mod tests {
 
         let cfg = CrossToml {
             targets: target_map,
-            build: None,
+            build: CrossBuildConfig::default(),
         };
 
         let test_str = r#"
-          [target.aarch64-unknown-linux-gnu]
-          volumes = ["VOL1_ARG", "VOL2_ARG"]
-          passthrough = ["VAR1", "VAR2"]
-          xargo = false
-          image = "test-image"
+            [target.aarch64-unknown-linux-gnu.env]
+            volumes = ["VOL1_ARG", "VOL2_ARG"]
+            passthrough = ["VAR1", "VAR2"]
+            [target.aarch64-unknown-linux-gnu]
+            xargo = false
+            image = "test-image"
         "#;
-        let parsed_cfg = CrossToml::from_str(test_str)?;
+        let (parsed_cfg, _) = CrossToml::parse(test_str)?;
```

Comment:
Should this also assert that the unused set is empty?

Reply:
good call, fixed

Reply:
should probably do it everywhere in these tests and not just this specific one :D